packages/cli/lang/en.json

```diff
@@ -14,7 +14,7 @@
   "commands_build_keypressListener_exit": "Exit: [CTRL + C], [ESC], or [Q]",
   "commands_build_keypressListener_watching": "Watching",
   "commands_build_options_h": "Show usage information",
-  "commands_build_options_m": "Path to the Polywrap manifest file (default: {default})",
+  "commands_build_options_m": "Path to the Polywrap Build manifest file (default: {default})",
```

Comment:
This was missing for some reason

Reply:
This was actually correct before. It's the path to the `polywrap.yaml` manifest, not `polywrap.build.yaml`.